config/galaxy.ini.sample

```diff
@@ -1333,7 +1335,7 @@ use_interactive = True
 # actual user instead of as the user running the Galaxy server process.  For
 # details on these options, see the documentation at:
 #
-# https://galaxyproject.org/admin/config/performance/cluster
+# https://galaxyproject.org/admin/config/performance/cluster#submitting-jobs-as-the-real-user
```

Comment:
Because of the redirect the anchor will not work, please use: https://docs.galaxyproject.org/en/latest/admin/cluster.html#submitting-jobs-as-the-real-user